compiler/vm/vmgen.nim

```diff
@@ -2768,18 +2852,32 @@ proc gen(c: var TCtx; n: PNode; dest: var TDest; flags: TGenFlags = {}) =
     let s = n[namePos].sym
     genProcLit(c, n, s, dest)
   of nkChckRangeF, nkChckRange64, nkChckRange:
-    let
-      tmp0 = c.genx(n[0])
-      tmp1 = c.genx(n[1])
-      tmp2 = c.genx(n[2])
-    c.gABC(n, opcRangeChck, tmp0, tmp1, tmp2)
-    c.freeTemp(tmp1)
-    c.freeTemp(tmp2)
-    if dest >= 0:
-      gABC(c, n, whichAsgnOpc(n), dest, tmp0)
+    let tmp0 = c.genx(n[0])
+    # XXX: range checks currently always happen, even if disabled by the user.
+    #      Once the range check injection logic is a MIR pass, this should be
```

Comment:
```suggestion
    #      Once the range check injection logic is an MIR pass, this should be
```

Reply:
Hm, but "MIR" doesn't start with a vowel sound, no? (or at least, I haven't pronounced it like that)

Reply:
I say 'MIR' like one would 'IBM', sounding out each letter, so 'M' -> 'em'. It's not a big deal, that was an autopilot thing.

Reply:
Ah, okay, that I didn't consider. I'll change the `a` to an `an` then.